openmetadata-ui/src/main/resources/ui/src/pages/TeamsPage/TeamsPage.tsx

```diff
@@ -217,7 +219,6 @@ const TeamsPage = () => {
     try {
       const data = await getTeamByName(name, {
         fields: [
-          TabSpecificField.USERS,
```

Comment:
I believe we can also remove this user field from the `fetchTeamAdvancedDetails` function. Can you verify it once ?

Reply:
Verified and removed the field from the function.